content/posts/two-factor-authentication.md

```diff
@@ -0,0 +1,20 @@
+title: Two-factor authentication is here!
+date: November 8, 2024
+description: We're excited to announce that two-factor authentication is now available for all Read the Docs users.
+category: Feature announcement
+tags: authentication, security
+authors: Santos Gallegos
+status: published
+image: /images/2fa.jpg
+image_credit: Photo by <a href="https://unsplash.com/@christianw?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Christian Wiediger</a> on <a href="https://unsplash.com/photos/shallow-focus-photography-of-love-lock-6t8PMIfqNVg?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+We're excited to announce that two-factor authentication (2FA) is now available for all Read the Docs users!
+
+Two-factor authentication adds an extra layer of security to your account by requiring a second form of verification in addition to your password.
```

Comment:
We should probably note what types of authentication is provided. It's just the 6 digit code currently, right?